spec/system/task_spec.rb

```diff
@@ -84,9 +83,7 @@
       end
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
-        # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = create(:task, :done_yesterday)
```

Comment:
完了日で場合分けしたテストケースがある訳でもないので、もっと簡潔にしても良いかもしれません。
※ 細かい部分なので修正しなくても大丈夫です！
```suggestion
        task = create(:task, :done)
```